email_reply_parser/__init__.py

```diff
@@ -282,6 +282,14 @@ def clean_email_content(body, include: Optional[bool], word_limit):
             # Remove the sign-off
             body = EmailMessage.remove_signoff(body, signoff_matches, word_limit)
 
+        # Split the body into lines and remove the last line if it matches any of the SIG_REGEX
+        # Use the unidecode copy to check for matches but remove lines from original body
+        body_lines_unidecode = body_clean.split('\n')
+        body_lines = body.split('\n')
+
+        if SIG_REGEX.match(body_lines_unidecode[-1]):
```

Comment:
I noticed a few cases in that anomaly where the sent from is on the first line as well. Maybe just also add a `if SIG_REGEX.match(body_lines_unidecode[0]): ...`

Reply:
I would also add a test for this kind of thing